circleguard/utils.py

```diff
@@ -110,3 +110,27 @@ def span_to_list(span):
         else:
             ret.add(int(s))
     return ret
+
+def convert_ur(ur, mods, *, to):
```

Comment:
missing doc for `to`. also `to` should have a default assigned imo

Reply:
disagree on the default, `convert_ur` is already a direction-neutral statement, having a call of `convert_ur(10)` without the `to` argument is hard to understand

Reply:
ok makes sense, but why is it a forced keyword only argument then? `convert_ur(10, mods, "cv")` looks fine to me

Reply:
ambiguous between `to` or `from` (am I starting as cv or ending as cv?)